caproto/server/common.py

```diff
@@ -1095,6 +1095,13 @@ async def tcp_handler(self, client, addr):
             while True:
                 try:
                     await circuit.recv()
+                except Exception:
```

Comment:
I think this will full mask the exception below?  I think these need to be re-ordered.

Reading commit-by-commit so you might have already fixed this in the future.

Reply:
yes you did.